src/Select/common/NoMatch.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import propTypes from 'prop-types'
+import { colors, spacers, theme } from '../../theme.js'
+
+const NoMatch = ({ filter }) => (
+    <div>
+        Nothing found for &quot;{filter}&quot;
```

Comment:
You're right. See also the top comment, we could accept a user supplied component here for example. The string's in there because it was in the specs, but as I understand from @varl we don't want translations in ui-core.

Reply:
No translations in ui-core.

Using ui-core components we should be able to satisfy the design specs, but for example, no design specs has any localised strings in languages other than English, so if we would be slaves to the specs we wouldn't have to support i18n anywhere. ;)